package.json

```
@@ -24,8 +24,8 @@
   ],
   "scripts": {
     "build": "npm run tsc.scripts && npm run tsc.prod && npm run rollup.prod.ci",
-    "build.esbuild": "npm run clean && npm run build && node scripts/build/esbuild/build.js",
-    "build.esbuild.watch": "npm run clean && npm run build && node scripts/build/esbuild/build.js --watch",
+    "build.esbuild": "npm run clean && npm run tsc.scripts && npm run tsc.prod && node scripts/build/esbuild/build.js",
+    "build.esbuild.watch": "npm run clean && npm run tsc.scripts && npm run tsc.prod && node scripts/build/esbuild/build.js --watch",
```

I wonder if we can pass in the watch flag in some way like this:
```suggestion
    "build.esbuild.watch": "npm run build.esbuild -- --watch",
```

and have it attached to the last command.

---

ohh seems like that does work, nice!